### PR TITLE
Bump up sqs worker templates to Node 16

### PR DESCRIPTION
**Pull request by samchungy**
It's finally here......

### CODE REVIEW

**Comment by 72636c** [comment]
Truly a momentous occasion

File: template/lambda-sqs-worker/serverless.yml [hunk 1]
```diff
@@ -25,11 +25,10 @@ provider:
   logRetentionInDays: 30
   name: aws
   region: ap-southeast-2
-  runtime: nodejs14.x
+  runtime: nodejs16.x
   architecture: arm64
   stackName: ${self:service}
   stage: ${env:ENVIRONMENT}
-  versionFunctions: true
```
Comment (samchungy) [inline template/lambda-sqs-worker/serverless.yml:32]: This is by default true

**Comment by 72636c** [comment]
Can we upgrade the CDK template too?

**Comment by samchungy** [comment]
> Can we upgrade the CDK template too?

Awaiting this to be released :( https://github.com/aws/aws-cdk/pull/20261

**Review by 72636c — APPROVED** [review]
Great stuff!

**Pull request merged**